plugins/vendor/zoom/ZoomDropFolderPlugin/batch/processors/zoomChatFilesProcessor.php

```diff
@@ -40,7 +40,8 @@ public function handleChatRecord($entry, $recording, $chatDownloadUrl)
 		KBatchBase::impersonate($entry->partnerId);
 		$attachmentAsset = $this->createAttachmentAssetForChatFile($recording->meetingMetadata->meetingId, $entry, $attachmentPlugin);
 		$attachmentAssetResource = new KalturaUrlResource();
-		$attachmentAssetResource->url = $this->getZoomRedirectUrlFromFile($recording);
+		list($redirectUrl, $urlHeaders) = $this->getZoomRedirectUrlFromFile($recording);

```

Comment:
you are now not putting the access token on the url but also not passing the headers to the apis